GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -348,6 +364,65 @@ public InstanceData parseInstance(Sheet sheet, int startRow,
                             }
                             break;
 
+                        case OPTION:
+                            // Two different possible formats:
+                            // With codes: code1:val1|code2:val2|...
+                            // Without codes: val1|val2|...
+                            String optionString = ExportImportUtils.parseCellAsString(cell);
+                            if (optionString.isEmpty()) {
+                                break;
+                            }
+                            String[] optionParts = optionString.split("\\|");
+                            List<Map<String, Object>> optionList = new ArrayList<>();
+                            for (String optionNode : optionParts) {
+                                String[] codeAndText = optionNode.split(":");
```

Comment:
Perhaps here you can add the split limit just to take into account erroneous input like `code:|` or `:text|`?